### PR TITLE
Add package.json to add dependence of qunit

### LINKED ISSUE #8: Cannot find module `qunit` when run node.js tests.js
We may need a package.json to set the qunit dependency.

### PR DESCRIPTION
**Pull request by pancx**
Add package.json to install qunit by npm install
resolve #8

### CODE REVIEW

File: modules/js/test/package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "bin",
```
Comment (huningxin) [inline modules/js/test/package.json:2]: change name to "opencv.js tests"?

File: modules/js/test/package.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  "description": "Tests for opencv js bindings",
+  "version": "1.0.0",
+  "dependencies" : { 
+    "qunit" : "latest"
+  },
+  "main": "opencv.js",
```
Comment (huningxin) [inline modules/js/test/package.json:8]: Can we remove this line? As we don't use main, right?

File: modules/js/test/package.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  "scripts": {
+    "test": "node tests.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/huningxin/opencv.git"
```
Comment (huningxin) [inline modules/js/test/package.json:14]: Please change the URL to "https://github.com/opencv/opencv.git"

File: modules/js/test/package.json [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  },
+  "keywords": [],
+  "author": "",
+  "license": "BSD-4-Clause",
+  "bugs": {
+    "url": "https://github.com/huningxin/opencv/issues"
```
Comment (huningxin) [inline modules/js/test/package.json:20]: ditto.

File: modules/js/test/package.json [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  },
+  "homepage": "https://github.com/huningxin/opencv"
```
Comment (huningxin) [inline modules/js/test/package.json:22]: ditto.

**Review by huningxin — CHANGES_REQUESTED** [review]

File: modules/js/test/package.json (hunk 1 above)
Comment (pancx) [reply modules/js/test/package.json:2]: I see the name should be the current directory name

File: modules/js/test/package.json (hunk 2 above)
Comment (pancx) [reply modules/js/test/package.json:8]: That's ok

File: modules/js/test/package.json (hunk 3 above)
Comment (pancx) [reply modules/js/test/package.json:14]: ok

File: modules/js/test/package.json (hunk 1 above)
Comment (huningxin) [reply modules/js/test/package.json:2]: I don't think it should be the current directory name. But name cannot use "." and "space". Let's use "opencv_js_tests". Please try that.

Comment (pancx) [reply modules/js/test/package.json:2]: fine

**Comment by huningxin** [comment]
@pancx , why did you close this PR?

**Comment by pancx** [comment]
@huningxin  I pull a new request

**Comment by huningxin** [comment]
you can just update this PR.

**Pull request closed without merging**